### PR TITLE
feat: parachain

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

**Review by jak-pan — APPROVED** [review]

**Review by jak-pan — CHANGES_REQUESTED** [review]
We would need to

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -424,7 +424,8 @@ parameter_types! {
 	pub NativeTokens: BTreeSet<(Vec<u8>, MultiLocation)> = {
 		let mut t = BTreeSet::new();
 		t.insert(("HDT".into(), MultiLocation::X2(Junction::Parent, Junction::Parachain { id: 200})));
```
Comment (jak-pan) [inline runtime/src/lib.rs:426]: We sure this is right?

Comment (enthusiastmartin) [reply runtime/src/lib.rs]: It does not have to be there. Already removed.

**Comment by lumir-mrkva** [comment]
moved to the https://github.com/galacticcouncil/basilisk-node

**Pull request closed without merging**